.travis.yml

```diff
@@ -0,0 +1,44 @@
+---
+sudo: false
+dist: trusty
+language: ruby
+cache: bundler
+before_install:
+  - bundle -v
+  - rm -f Gemfile.lock
+  - gem update --system
+  - gem --version
+  - bundle -v
+script:
+  - 'bundle exec rake $CHECK'
+  - 'bundle exec rspec spec/'
+bundler_args: --without system_tests
+rvm:
+  - 2.5.0
+env:
+  global:
+    - BEAKER_PUPPET_COLLECTION=puppet6 PUPPET_GEM_VERSION="~> 6.0"
+matrix:
+  fast_finish: true
+  include:
+    -
+      env: CHECK="syntax lint metadata_lint check:symlinks check:git_ignore check:dot_underscore check:test_file rubocop"
+    -
+      rvm: 2.4.4
+    -
```

Comment:
This re-runs rspec with Ruby 2.5. That's already happening at line 24 since you made every matrix entry run specs.